File: LUA_esimerkit/u_bark_Heli.lua
bark = {
  authors = {'hatusta'},

  const= {
    category = 'model',
    priority = '--',
    retval1 = {'B', u = 'mm'},  -- kuorenpaksuus 
    impl = function()
      return 5
    end,
  },
  
  d_ub = {
    category = 'model',
    retval1 = {'d_ub', u = 'mm'},
    param1 = {'d', u = 'mm'},
    param2 = {'B', u = 'mm'},
    impl = function(d,b)
      return d-b
    end,
  },

d_ub_h = {
    category = 'model',
    retval1 = {'d_ub_h', u = 'mm'},
    param1 = {'d_h', u = 'mm'},
    param2 = {'B', u = 'mm'},
    impl = function(d,b)
      return d-b
    end,
  }
}  

File: R_examples/u_bark_Heli.R
# @unit b mm
# @return b kuorenpaksuus
const <- function()
{
  return(5)
}

# @param d
# @param b
# @unit d mm
# @unit b mm
# @unit d_ub mm
# @return d_ub
d_ub <- function(d,b) 
{
  reutrn(d-b)
}

# @param d_h
# @param b
# @unit d_h mm
# @unit b mm
# @unit d_ub_h mm
# @return d_ub_h
d_ub_h <- function(d,b)
{
  return(d-b)
}
